pipeline/5/Jenkinsfile-tier-0.groovy

```diff
@@ -10,7 +10,6 @@ def testStages = ['deploy': {
                     stage('Deployment suite') {
                         script {
                             withEnv([
-                                "sutVMConf=conf/inventory/rhel-8.3-server-x86_64.yaml",
```

Comment:
I prefer having this option available to the test suite... we can chose the default if it's not available